instance/management/commands/find_orphan_dbs.py

```diff
@@ -0,0 +1,196 @@
+"""
+import getpass
+import logging
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db import connections
+import MySQLdb as mysql
+import pymongo
+
+from instance.models.database_server import MongoDBServer, MySQLServer
+
+LOG = logging.getLogger(__name__)
+
+MONGO_SYSTEM_DBS = {"admin", "local", "config"}
+MYSQL_SYSTEM_DBS = {"information_schema", "performance_schema", "sys"}
+
+class Command(BaseCommand):
+    """
+    find_orphan_dbs management command class
+    """
+    help = 'Finds orphaned databases and generates commands to drop them.'
+
+    def add_arguments(self, parser):
+
+        subparsers = parser.add_subparsers(
+            title='db type',
+        )
+        mongo_parser = subparsers.add_parser('mongo', help='Find orphaned MongoDB databases')
+        mongo_parser.add_argument(
+            '--host',
+            help="MongoDB host to connect to",
+            type=str,
+            default='localhost'
+        )
+        mongo_parser.add_argument(
+            '--port',
+            help="MongoDB port to connect to",
+            type=int
+        )
+        mongo_parser.add_argument(
+            "-u",
+            "--user",
+            help="User to authenticate as",
+            type=str
+        )
+        mongo_parser.add_argument(
+            "-p",
+            "--prompt_password",
+            action='store_true',
+            help="Prompt for password",
+        )
+        mongo_parser.add_argument(
+            '--exclude-dbs',
+            help=(
+                'MongoDB databases to ignore when checking for orphans.\n'
+                'System dbs are always excluded.'
+            ),
+            nargs='+',
+            type=str,
+            default=set()
+        )
+        mongo_parser.set_defaults(func=self.find_mongo_orphans)
+
+        mysql_parser = subparsers.add_parser('mysql', help='Find orphaned MySQL databases')
+        mysql_parser.add_argument(
+            '--exclude-dbs',
+            help=(
+                'MySQL databases to ignore when checking for orphans.\n'
+                'System dbs are always excluded.'
+            ),
+            nargs='+',
+            type=str,
+            default=set()
+        )
+        mysql_parser.add_argument(
+            '--host',
+            help="Host to connect to",
+            type=str,
+            default='localhost'
+        )
+        mysql_parser.add_argument(
+            '--port',
+            help="Port to connect to",
+            type=int
+        )
+        mysql_parser.add_argument(
+            "-u",
+            "--user",
+            help="User to authenticate as",
+            type=str
+        )
+        mysql_parser.add_argument(
+            "-p",
+            "--prompt_password",
+            action='store_true',
+            help="Prompt for password",
+        )
+        mysql_parser.set_defaults(func=self.find_mysql_orphans)
```

Comment:
We [store](https://manage.opencraft.com/admin/instance/mysqlserver/7/change/) all this information in database as fields of `MySQLServer` model. Therefore, probably we could reduce this just to one argument, i.e. `mysqlserver_id`?

Reply:
I've considered that but it decided against it because it assumes all `MySQLServer` instances are registered in OCIM...which I now realize, from your other comment, is probably true...
In fact, we probably don't even need `mysqlserver_id` because we can just go through all of them using `MySQLSerevr.objects.get()`, so I'll remove all of these.